src/main/java/Barry.java

```diff
@@ -0,0 +1,322 @@
+            super(name);
+        }
+
+        // Method which returns details of the object as a String
+        @Override
+        public final String detailsAsString() {
+            return "[T][" + (this.getStatus() ? "X" : " ") + "] " + this.getName();
+        }
+    }
+
+    // A class that inherits from the Task class, for creating Deadlines
+    // (tasks that need to be done before a specific date/time)
+    private static class Deadline extends Task {
+        private final String deadlineDateTime;
+
+        // Constructor for Deadlines, initialised with the name of the task and the deadline
+        public Deadline(String name, String deadline) {
+            super(name);
+            this.deadlineDateTime = deadline;
+        }
+
+        // Method which returns details of the object as a String
+        @Override
+        public final String detailsAsString() {
+            String basicInfo = "[D][" + (this.getStatus() ? "X" : " ") + "] " + this.getName();
+            return basicInfo + " (by: " + deadlineDateTime + ")";
+        }
+    }
+
+    // A class that inherits from the Task class, for creating Events
+    // (tasks that start and end at a specific date/time)
+    private static class Event extends Task {
+        private final String eventStartDateTime;
+        private final String eventEndDateTime;
+
+        // Constructor for Events, initialised with the name of the task, as well as the start and end of the event
+        public Event(String name, String start, String end) {
+            super(name);
+            this.eventStartDateTime = start;
+            this.eventEndDateTime = end;
+        }
+
+        // Method which returns details of the object as a String
+        @Override
+        public final String detailsAsString() {
+            String basicInfo = "[E][" + (this.getStatus() ? "X" : " ") + "] " + this.getName();
+            return basicInfo + " (from: " + eventStartDateTime + " to: " + eventEndDateTime + ")";
+        }
+    }
+
+    // ArrayList for storing user-added tasks.
+    private static final ArrayList<Task> taskList = new ArrayList<>();
+
+    // Greets the user
+    private static void greet() {
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\t Greetings! I'm Barry.\n" + "\t How could I assist you?");
+        System.out.println("\t [Type help for a list of available commands]");
+        System.out.println("\t____________________________________________________________\n");
+    }
+
+    // Exits the program with a message
+    private static void exit() {
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\t Bye, see you next time! :)");
+        System.out.println("\t____________________________________________________________\n");
+    }
+
+    // List all available commands
+    private static void help() {
```

Comment:
Like the help command! Nice! However, maybe instead of manually adding space, consider adding `\t`